clustering/k-means-clustering-plugin/README.md

```diff
@@ -51,13 +56,16 @@ For more information on WIPP, visit the [official WIPP page](https://isg.nist.go
 
 ## Options
 
-This plugin takes four input argument if methods other than 'Manual' is selected else three input arguments and one output argument:
+This plugin takes eight input arguments and one output argument:
 
 | Name             | Description                                                                 | I/O    | Type          |
 | ---------------- | --------------------------------------------------------------------------- | ------ | ------------- |
-| `--inpdir`       | Input csv collection                                                        | Input  | csvCollection |
+| `--inpdir`       | Input tabular data                                                          | Input  | genericData   |
+| `--filePattern`  | Pattern to parse tabular files                                              | Input  | string        |
 | `--methods`      | Select either Elbow or Calinski Harabasz or Davies Bouldin or Manual method | Input  | enum          |
 | `--minimumrange` | Enter minimum k-value                                                       | Input  | integer       |
 | `--maximumrange` | Enter maximum k-value                                                       | Input  | integer       |
 | `--numofclus`    | Enter number of clusters                                                    | Input  | integer       |
-| `--outdir`       | Output collection                                                           | Output | csvCollection |
+| `--fileExtension`| File Format of output tabular files                                         | Input  | string        |
```

Comment:
This should not be an input. We should only really support csv and arrow (feather), and only export in csv and arrow. The output should be controlled by the environment variable, not an input.